owncloudData/src/main/java/com/owncloud/android/data/storage/LocalStorageProvider.kt

```diff
@@ -86,6 +89,52 @@ sealed class LocalStorageProvider(private val rootFolderName: String) {
     @SuppressLint("UsableSpace")
     fun getUsableSpace(): Long = getPrimaryStorageDirectory().usableSpace
 
+    /**
+     * Checks if there is user data which does not have a corresponding account in the Account manager.
+     */
+    private fun getDanglingAccountDirs(context: Context?): List<File> {
+        val am = AccountManager.get(context)
+        val rootFolder = File(getRootFolderPath())
+        val danglingDirs = mutableListOf<File>()
+        for (dir in rootFolder.listFiles()) {
+            var dir_is_ok = false
```

Comment:
In Kotlin we try to use underscores only for constants -> https://kotlinlang.org/docs/coding-conventions.html#property-names

Reply:
thx